tests/e2e/travis-e2e.sh

```diff
@@ -13,16 +13,36 @@ chmod +x kubectl
 curl -Lo kind https://github.com/kubernetes-sigs/kind/releases/download/v0.6.1/kind-linux-amd64
 chmod +x kind
 
-./kind create cluster --image=kindest/node:v1.17.0
-export KUBECONFIG="$(./kind get kubeconfig-path)"
+run_e2e_tests() {
+    cluster_version=$1
 
-# create namespace, permissions, and CRDs
-./kubectl create -f manifests/setup
+    ./kind create cluster --image=kindest/node:$cluster_version
```

Comment:
This spins up multiple versions of k8s in the current master e2e to test against? I meant we need to change the versions of the cluster that is spun up in their respective release branches, not in master.

Reply:
In master it will only use `v1.17.0` as specified here:
```
compatibility_matrix["master"]="v1.17.0"
```
I was thinking about cherry-picking this commit to release-0.3 and release-0.4. I felt that this version would be more convenient when creating a new release but it might be better to update the script inside of each branch. wdyt?

Reply:
Don't think the e2e script in master needs to know about each release branch, only current version of k8s we are testing against.

> but it might be better to update the script inside of each branch

Yes just updating the script in each release branch and reflecting those number of k8s versions we are testing against in the README is what I meant. :)